formulas/glance/configure.sls

```diff
@@ -74,3 +74,44 @@ glance_api_service:
     - enable: true
     - watch:
       - file: /etc/glance/glance-api.conf
+
+download_image_bakery:
```

Comment:
Can you change this from a cmd.run to a git.latest?  You may also have to modify install to include the git package (it's present in most places already though, so you may not have to)